src/utils.ts

```diff
@@ -471,6 +471,11 @@ export function getRuntimeConfig(config: Partial<IUserConfig> = {}): IRuntimeCon
         }
     }
 
+    // The webRoot config is equivalent to a pathMapping entry of { '/': '${webRoot}' }.
+    if (!pathMapping.hasOwnProperty('/')) {
```

Comment:
This isn't working for me because I'm getting a pathMapping object back from the `settings.get('pathMapping')` call that already has `{'/': '${workspaceFolder}'}`. However, I don't have that set in my settings.json file so I'm still trying to figure out where it's coming from.

@mliao95 any idea what might be causing this?

Reply:
Not off the top of my head.  I'll dig into it

Reply:
Sorry - got sidetracked, not sure if you already figured it out, but the default value for `pathMapping` is `{'/': '${workspaceFolder}'}` as defined in the `package.json`:
https://github.com/microsoft/vscode-edge-devtools/blob/main/package.json#L217

Reply:
In that case I think we should always give precedence to `webRoot`. The default `webRoot` is `{$workspaceFolder}`, so this will not affect the default `pathMapping` unless the user has specified something else for `webRoot`.

Reply:
Thanks @mliao95. Based on the issue report, I think user expectations are that existing launch configs with `webRoot` properties just work with our extension the way they do for vscode JS debugging sessions. On top of that, the launch config should take priority over the extension setting because any logic on the vscode-js-debug side won't even be aware of our extension settings.